repository-data/webfiles/src/main/resources/site/freemarker/common/base-layout.ftl

```diff
@@ -88,11 +88,22 @@
           </div>
         </div><!--
         --><div class="layout__item layout-2-3">
+          <#if hstResponseChildContentNames?seq_contains("tabs") >
```

Comment:
Small that this `if` block is, I think it warrants turning into a macro, which could then be applied here (lines 91-95) and in lines 102-106.

Reply:
Will do